### PR TITLE
KOGITO-3758 Replace maven-remote-resources-plugin with copy resources

### PR DESCRIPTION
**Pull request by cristianonicolai**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: integration-tests/integration-tests-trusty-service/integration-tests-trusty-service-springboot/src/main/resources/TrafficViolation.dmn [hunk 1]
```diff
@@ -0,0 +1,230 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (kostola) [inline integration-tests/integration-tests-trusty-service/integration-tests-trusty-service-springboot/src/main/resources/TrafficViolation.dmn]: Isn't this file supposed to be copied from the common module by the plugin you're configuring? Maybe it was added by mistake?

**Comment by kostola** [comment]
I tried to use "copy resources" also in the PR where I introduced these integration tests, but I had a weird error that made me stick with the original plugin.
Now I can't tell exactly what is different from my attempt, but I tried to build this branch and that error is not there anymore.
I will approve the PR once [my comment](https://github.com/kiegroup/kogito-apps/pull/518#discussion_r517219381) is resolved.

File: integration-tests/integration-tests-trusty-service/integration-tests-trusty-service-springboot/src/main/resources/TrafficViolation.dmn (hunk 1 above)
Comment (cristianonicolai) [reply integration-tests/integration-tests-trusty-service/integration-tests-trusty-service-springboot/src/main/resources/TrafficViolation.dmn]: @kostola my bad, the file was added by mistake

**Review by kostola — APPROVED** [review]
LGTM :+1:

**Review by tiagodolphine — APPROVED** [review]

**Pull request merged**